src/rapport/03_resultaten.Rmd

```diff
@@ -6,22 +6,119 @@ bibliography: moneos.bib
 
 ## Inschatting tonnage droge massa macrozoöbenthos (totaal + foutenmarge)
 
+```{r benthos-afdw}
+benthos <- read_vc(file = "benthos",
+                   root = find_root_file("data",
+                                         criterion = is_git_root))
+
+benthos_afdw <- benthos %>%
+  mutate(survey = ifelse(grepl("raai", .$campagne.y), "raai", "spatial")) %>%
+  group_by(survey, campagne.x, campagne.y, staal, staalcode, jaar, datum.x,
+           datum.y, real_X, real_Y, Z, geomorf, ecotoop_werkelijk,
+           ecotoop_gepland, SalZone, Vallei_deel, Omessegment, KRWzone) %>%
+  summarise(tot_aantal = sum(aantal),
+            tot_afdw = sum(AFDW),
+           .groups = "drop")
+```
+
 ```{r eval=FALSE}
-# data.frame maken met alle survey design variabele en responsvariabelen
-srv_data <- data.frame()
+# mogelijke fouten in de data:
+benthos_afdw %>%
+  filter(str_extract(campagne.x, "SP\\d{2}") !=
+           str_extract(campagne.y, "SP\\d{2}")) %>%
+  select(staal, survey, campagne.x, campagne.y, jaar, datum.x, datum.y)
+# verschillend jaartal in campagne.x en campagne.y
+# variabele jaar komt overeen met campagne.y en is conform datum.x en datum.y
+# dus best variabelen jaar en survey gebruiken (niet campagne.x)
```

Comment:
Staal DI_SP_04 staat voor beide jaren twee keer in de metadata met lichtjes
andere X-Y coordinaten en ander ecotoop_werkelijk. 
--> deze staal zit ook nog dubbel in deze dataset. Zal het gemiddelde tonnage niet beïnvloeden maar mogelijk wel de spreiding.

Reply:
Dat is een staal dat in één van de volgende filter stappen wegvalt omdat het niet tot Zeeschelde of Durme behoort. Voor onze oefening dus geen probleem, maar best toch navragen wat hiermee aan de hand is.

Reply:
Ik merkte ook dat er twee stalen waren met negatieve AFDW: GSc_SP_02 (2018) en  KD19_16 (2019). Ik heb [hier](https://docs.google.com/document/d/1Pmnnruqs16YFFD9H9kHbPYUbdgzb9JKnKz75m7I7pbI/edit) enkele vragen opgeschreven die we nog moeten stellen aan de kenners.